### PR TITLE
pk-main: Mark some early-start error messages as translatable

### PR DESCRIPTION
**Pull request by pwithnall**
As suggested by Matthias on
https://github.com/PackageKit/PackageKit/pull/559#issuecomment-1173005774

Signed-off-by: Philip Withnall <pwithnall@endlessos.org>

### CODE REVIEW

File: src/pk-main.c [hunk 1]
```diff
@@ -165,13 +165,13 @@ main (int argc, char *argv[])
 	conf = g_key_file_new ();
 	conf_filename = pk_util_get_config_filename ();
 	if (conf_filename == NULL) {
-		g_printerr ("Config file was not found.\n");
+		g_printerr (_("Config file was not found.\n"));
```
Comment (hughsie) [inline src/pk-main.c:168]: I don't think the translators like newlines; e.g. I think you want:

g_printerr ("%s\n", _("Config file was not found."));

File: src/pk-main.c [hunk 2]
```diff
@@ -196,7 +196,7 @@ main (int argc, char *argv[])
 	if (backend_name == NULL || g_strcmp0 (backend_name, "auto") == 0) {
 		ret  = pk_util_set_auto_backend (conf, &error);
 		if (!ret) {
-			g_printerr ("Failed to resolve auto: %s\n", error->message);
+			g_printerr (_("Failed to resolve auto: %s\n"), error->message);
```
Comment (hughsie) [inline src/pk-main.c:199]: same:

    g_printerr ("%s: %s\n", _("Failed to resolve auto"), error->message);

and TRANSLATOR comment please.

Comment (pwithnall) [reply src/pk-main.c]: Can’t use a format string `%s: %s` as the translators might want to quote the error message or add text after it.

I’ve used an intermediate variable

Comment (hughsie) [reply src/pk-main.c]: I don't think we've done that in other places tho, right?

Comment (pwithnall) [reply src/pk-main.c]: No reason to get it wrong here too

**Pull request merged**